docs/operator-manual/rbac.md

```diff
@@ -275,96 +381,21 @@ data:
+Additionally, it can be used against the live policy in the cluster your Argo CD is running in.
 
 ### Validating a policy
 
-To validate a policy stored in a local text file:
-
-```shell
-argocd admin settings rbac validate --policy-file somepolicy.csv
-```
-
-To validate a policy stored in a local K8s ConfigMap definition in a YAML file:
-
-```shell
-argocd admin settings rbac validate --policy-file argocd-rbac-cm.yaml
-```
-
-To validate a policy stored in K8s, used by Argo CD in namespace `argocd`,
-ensure that your current context in `~/.kube/config` is pointing to your
-Argo CD cluster and give appropriate namespace:
-
-```shell
-argocd admin settings rbac validate --namespace argocd
-```
+To check whether your new policy is valid and understood by Argo CD's RBAC implementation,
+you can use the [`argocd admin settings rbac validate` command](../user-guide/commands/argocd_admin_settings_rbac_validate.md).
 
 ### Testing a policy
 
 To test whether a role or subject (group or local user) has sufficient
 permissions to execute certain actions on certain resources, you can
-use the `argocd admin settings rbac can` command. Its general syntax is
-
-```shell
-argocd admin settings rbac can SOMEROLE ACTION RESOURCE SUBRESOURCE [flags]
-```
-
-Given the example from the above ConfigMap, which defines the role
-`role:org-admin`, and is stored on your local system as `argocd-rbac-cm-yaml`,
-you can test whether that role can do something like follows:
-
-```console
-$ argocd admin settings rbac can role:org-admin get applications --policy-file argocd-rbac-cm.yaml
-Yes
-
-$ argocd admin settings rbac can role:org-admin get clusters --policy-file argocd-rbac-cm.yaml
-Yes
-
-$ argocd admin settings rbac can role:org-admin create clusters 'somecluster' --policy-file argocd-rbac-cm.yaml
-No
-
-$ argocd admin settings rbac can role:org-admin create applications 'someproj/someapp' --policy-file argocd-rbac-cm.yaml
-Yes
-```
-
-Another example,  given the policy above from `policy.csv`, which defines the
-role `role:staging-db-admin` and associates the group `db-admins` with it.
-Policy is stored locally as `policy.csv`:
-
-You can test against the role:
-
-```console
-$ # Plain policy, without a default role defined
-$ argocd admin settings rbac can role:staging-db-admin get applications --policy-file policy.csv
-No
-
-$ argocd admin settings rbac can role:staging-db-admin get applications 'staging-db-project/*' --policy-file policy.csv
-Yes
-
-$ # Argo CD augments a builtin policy with two roles defined, the default role
-$ # being 'role:readonly' - You can include a named default role to use:
-$ argocd admin settings rbac can role:staging-db-admin get applications --policy-file policy.csv --default-role role:readonly
-Yes
-```
-
-Or against the group defined:
-
-```console
-$ argocd admin settings rbac can db-admins get applications 'staging-db-project/*' --policy-file policy.csv
-Yes
-```
```

Comment:
I don't think that we should remove the examples from the doc.

Reply:
duplicated from the CLI command examples and replaced by the link to the CLI command

Reply:
Ok.. in this case, please clarify in the doc that they can refer to the CLI command doc to see usage examples.